manifests/setup.pp

```diff
@@ -46,10 +46,6 @@
     default   => $script_path
   }
 
-  File {
-    backup => false,
-  }
```

Comment:
I couldn't come up with a way to get the value of $backup inside concat::setup short of making it a parameterized class. Plus it seems that this is being inherited anyway